specification/containerservice/resource-manager/readme.md

```diff
@@ -400,6 +401,19 @@ input-file:
 
 ```
 
+### Tag: package-2022-02-02-preview-only
```

Comment:
Why define this tag since there is already tag package-preview-2022-02? If you want to add directive for python SDK, you can add it in readme.python.md

Reply:
Hi @chunyu3, for a long time, container service has always had two sets of api tags, normal (package-xxx) and only (package-xxxx-only).

For the normal one (which is also the default tag and used by java/golang), it should also include several sets of old swagger (e.g. 2017-07-01/containerService) and I just fixed this in commit [#87c80be](https://github.com/Azure/azure-rest-api-specs/pull/17889/commits/87c80bef6b8d1c64460350e57520f77cd6b7919a).

For the only one (which has directive definition and only used by python), it only includes the new swagger (e.g. xxxx/managedCluster).

Could you please help take another review? Thanks!